tests/test.cpp

```diff
@@ -1809,6 +1815,10 @@ int main(int /*argc*/, const char * /*argv*/[]) {
   #else
     auto &flatbuf = flatbuf1;
   #endif // !defined(FLATBUFFERS_CPP98_STL)
+#ifndef FLATBUFFERS_CPP98_STL
```

Comment:
can you please use the same pre-processor indentation style as the rest of the file.